superset-frontend/src/components/DvtRadio/index.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import {
+  StyledRadio,
+  StyledRadioCheck,
+  StyledRadioLabel,
+} from './dvt-radio.module';
+
+export interface DvtRadioProps {
+  label: string;
+  value: string;
+  active: string;
+  setActive: (active: string) => void;
+}
+
+const DvtRadio: React.FC<DvtRadioProps> = ({
+  label,
+  value,
+  active,
+  setActive,
+}) => {
+  const handleRadioClick = () => {
+    setActive(value);
+  };
+
+  return (
+    <StyledRadio onClick={handleRadioClick}>
```

Comment:
setActive(value); sadece tek olursa onClick içinde () => setActive(value); verelim yeterli

gerisi const handleRadioClick kaldıralım. Her zaman bi tane function varsa onClick kalsın minify için